### DISCUSSION TITLE
请问一下，登录界面老是显示network error  是怎么回事？

### DISCUSSION THREAD

**Discussion by jimmyhjbo** [ZuodaoTech/everyone-can-use-english#843] [Q&A] [answered]
我用的是windows的版本，昨天还可以登录的今天登录界面显示network errors。用不了软件。

**Comment by jimmyhjbo** [comment]
![Uploading mmexport1721399679621.png…]()

**Comment by SkyPan19** [comment]
我也是这样子，刚刚试了，现在还是登不进去

**Comment by weiwuzhiyu** [comment]
同问，注册显示network error

**Comment by an-lee** [comment]
尝试科学上网吧

**Reply by jimmyhjbo** [reply]
试过开vpn也不行

**Comment by an-lee** [comment, marked as answer]
解决办法： https://github.com/ZuodaoTech/everyone-can-use-english/discussions/853#discussioncomment-10101808
